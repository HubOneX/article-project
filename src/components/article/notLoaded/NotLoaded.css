.not-loaded-info {
  text-align: center;
  padding: 15px;
  margin: 0 20px 20px;

  @media screen and (max-width: 480px) {
    padding: 15px 0;
    font-size: 13px;
  }
}
