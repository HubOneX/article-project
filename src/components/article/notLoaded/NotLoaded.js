import "./NotLoaded.css";

const NotLoaded = () => {
  return (
    <p className="not-loaded-info">
      Sorry, we couldn't load this article. Please try again later.
    </p>
  );
};

export default NotLoaded;
