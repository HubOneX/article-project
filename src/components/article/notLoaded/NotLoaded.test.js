import renderer from "react-test-renderer";
import NotLoaded from "./NotLoaded";

describe("NotLoaded", () => {
  it("should match the snapshot", () => {
    const notLoaded = renderer.create(<NotLoaded />);

    expect(notLoaded).toMatchSnapshot();
  });
});
