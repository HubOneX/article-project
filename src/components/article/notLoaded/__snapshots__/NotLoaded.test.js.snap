// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotLoaded should match the snapshot 1`] = `
<p
  className="not-loaded-info"
>
  Sorry, we couldn't load this article. Please try again later.
</p>
`;
